_data/CONTRIBUTORS.yaml

```diff
@@ -55,3 +55,24 @@ Shoaib Sufi:
     orcid: 0000-0001-6390-2616
     affiliation: The University of Manchester / Software Sustainability Institute
     role: editor
+Jason Maassen:
+    git: jmaassen
+    email: J.Maassen@esciencecenter.nl
+    role: editor
+    affiliation: Netherlands eScience Center
+Zhiming Zhao:
+    git: zmzhao
+    email: z.zhao@uva.nl
+    role: editor
+    affiliation: Universiteit van Amsterdam
+Christian Hüser:
+    git: christianhueserhzdr
+    affiliation: HZDR
+Azza Gamgami:
+    git: azza-max
+Kirsty Pringle:
+    git: KirstyPringle
+Esteban González:
+    git: esgg
+Giacomo Peru:
+    git: gperu
```

Comment:
Shouldn't we add institution/ORCID for the last contributors? Or is the idea that they update themselves